searx/network/client.py

```diff
@@ -191,7 +191,7 @@ def new_client(
         event_hooks = {'response': [hook_log_response]}
 
     return httpx.AsyncClient(
-        transport=transport,
+        # transport=transport,
```

Comment:
When using SearXNG's transport, all our requests will get blocked, probably due to fingerprinting. We should find an alternative solution because this is just a hacky change to make things work for now, but idk how ...
I've already spent a lot of time debugging to figure out that the 403 request forbidden responses are caused by the transport layer used.

Reply:
One has to set `enable_http2 = False` in the engine as global variable, wow...